### PR TITLE
Fixing the docs of LM for ASR models.

### PR DESCRIPTION
**Pull request by VahidooX**

### CODE REVIEW

File: docs/source/asr/asr_language_modelling.rst [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 
 It is possible to use both approaches on the same ASR model.
 
 
 .. _ngram_modelling:
 
+*************************
 N-gram Language Modelling
--------------------------
+*************************
```
Comment (ekmb) [inline docs/source/asr/asr_language_modelling.rst:17]: Modelling -> Modeling

File: docs/source/asr/asr_language_modelling.rst [hunk 2]
```diff
@@ -1,24 +1,27 @@
+#####################
 ASR Language Modeling
-=====================
+#####################
 
 Language models have shown to help the accuracy of ASR models. NeMo support the following two approaches to incorporate language models into the ASR models:
-    + N-gram Language Modelling
-    + Neural Rescoring
+
+* :ref:`ngram_modelling`
```
Comment (ekmb) [inline docs/source/asr/asr_language_modelling.rst:7]: `modelling` -> `modeling`

File: docs/source/asr/asr_language_modelling.rst [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
 
+*************************
 N-gram Language Modelling
--------------------------
+*************************
 
 In this approach, an N-gram LM is trained on text data, then it is used in fusion with beam search decoding to find the
-best candidates. The beam search decoders in NeMo support language models trained by KenLM library
-`https://github.com/kpu/kenlm <https://github.com/kpu/kenlm>`__.
+best candidates. The beam search decoders in NeMo support language models trained by KenLM library (
```
Comment (ekmb) [inline docs/source/asr/asr_language_modelling.rst:20]: `trained with` ?

Comment (VahidooX) [reply docs/source/asr/asr_language_modelling.rst:20]: fixed.

**Review by ekmb — APPROVED** [review]

**Pull request merged**